src/libsrx/src/common.h

```diff
@@ -25,6 +25,7 @@ int asprintf(char **strp, const char *fmt, ...);
 #define INFO(fmt, ...)  syslog(LOG_INFO, fmt, ##__VA_ARGS__)
 #define NOTE(fmt, ...)  syslog(LOG_NOTICE, fmt, ##__VA_ARGS__)
 #define WARN(fmt, ...)  syslog(LOG_WARNING, "%s: " fmt, __func__, ##__VA_ARGS__)
+#define ERR(fmt, ...)   syslog(LOG_ERR, fmt, ##__VA_ARGS__)
 #define ERROR(fmt, ...) syslog(LOG_ERR, "%s: " fmt, __func__, ##__VA_ARGS__)
```

Comment:
It feels like we'll end up with people just copying whatever definition was closest to the code they're changing that day.

I assume this is to match the output to messages created at the info/note levels?

Maybe we should just drop `__func__` from `WARN()` and `ERROR()` instead?

Reply:
Yeah these are a mess.  My take on it is that the debug macros, which need not just the correct logmask to be set but also a debug flag, they could carry the `__func__` and all warnings and errors (and notes) should be user-friendly formatted messages without function name.  Because we can usually identify the location, and when we can't (duplicate messages) we're missing line info anyway.

What do you think?

Reply:
I've just pushed an update with that suggestion (only `DEBUG()` has `__func__`).